tests/upgrade/legacy_to_postgres_run.sh

```diff
@@ -236,6 +242,62 @@ check_legacy_db_status() {
     test_equals_non_silent "$(echo "$dbStatus" | jq '.databaseType' -r)" "RocksDB"
 }
 
+restore_56_1_backup() {
+    info "Restoring a 56.1 backup"
```

Comment:
I am not familiar with what `56.1` is. Could you explain it?

Reply:
it is a backup from 3.56.1.  Since I'm moving that function specifically to here and only to be used here, I will make that clear.